docassemble/ALAutomatedTestingTests/data/questions/al_set_up_testing.yml

```
@@ -293,7 +293,7 @@ content: |
     
     An admin of a GitHub **repository** can set the secrets for that GitHub repository on the GitHub website itself.
     
-    A collaborator of a GitHub **repository** can set the secrets for the GitHub repository through **this tool**. A non-admin is unable to set secrets on GitHub itself. Yes, it's confusing. We don't know why GitHub works that way ¯\_(ツ)_/¯
+    A collaborator of a GitHub **repository** can set the secrets for the GitHub repository through **this tool**. A non-admin is unable to set secrets on GitHub itself. Yes, it's confusing. We don't know why GitHub works that way ¯\\_(ツ)_/¯
```

This actually didn't work for me. The only way I could get it to display with equally sized arms was with `¯\\\\_(ツ)\_/¯`

```suggestion
    A collaborator of a GitHub **repository** can set the secrets for the GitHub repository through **this tool**. A non-admin is unable to set secrets on GitHub itself. Yes, it's confusing. We don't know why GitHub works that way ¯\\\\_(ツ)\_/¯
```

---

That makes sense, I hadn't checked it, I just used what I know sometimes works in other markdown editors.